macros/compareprt.swb

```diff
@@ -0,0 +1,72 @@
+' COMPARE SOLIDWORKS PARTS
+' COPYRIGHT (C) 2021 HENRY FRANKS
+
+Option Explicit
+
+Declare PtrSafe Function lstrcpyn_long_string Lib "kernel32" Alias "lstrcpynA" (ByVal DestString As String, ByVal SourcePointer As Long, ByVal MaxLen As Long) As Long
+Declare PtrSafe Function lstrlen_long Lib "kernel32" Alias "lstrlenA" (ByVal SourcePointer As Long) As Long
+Declare PtrSafe Function GetCommandLine_long Lib "kernel32" Alias "GetCommandLineA" () As Long
+
+Sub main()
+
+    Dim Args()         As String
+    Dim swApp          As Object
+    Dim swUtil         As Object
+    Dim swUtilCompGeom As Object
+    Dim longStatus     As Long
```

Comment:
I like aligning like this, it's very aesthetically pleasing :ok_hand: btw, i have some auto-aligning vimscript if you want it